src/js/project.js

```diff
@@ -1,6 +1,20 @@
-const startApp = () => {
-  const body = document.querySelector('body');
-  body.innerHTML = '<h2>JAVASCRIPT ENABLED</h2>';
+import closeModal from './closemodal';
+import dynamicValues from './dynamicvalues';
+import dynamicId from './dynamicid';
+import addItem from './additems';
+import addItemToUi from './additemtoui';
+
+const create = e => {
+  e.preventDefault();
+  const { target } = e;
+  const values = target.querySelectorAll('.values');
+  const databaseName = target.dataset.databasename;
+  const formType = target.dataset.formtype;
```

Comment:
I think you can destructure this directly.

Reply:
I'll check it and make necessary adjustments